src/classic/element/ReactElementValidator.js

```diff
@@ -247,6 +247,18 @@ function checkPropTypes(componentName, propTypes, props, location) {
       } catch (ex) {
         error = ex;
       }
+      warning(
+        !error || error instanceof Error,
+        '%s: the type of %s `%s` is invalid; the type checker function ' +
```

Comment:
Maybe "type checker … is invalid" or "type specification … is invalid"? I could see misinterpreting this to mean that an invalid value was passed.